allocations/tasks.py

```diff
@@ -346,14 +390,16 @@ def check_charge():
             openstack_endtime = openstack_r.get("end_time")
             portal_endtime = portal_r.get("end_time").strftime(utils.DATETIME_FORMAT)
             if openstack_endtime != portal_endtime:
-                LOG.error(
-                    f"{resource_id} at {region} has incorrect end time! "
-                    f"openstack={openstack_endtime}, portal={portal_endtime}"
-                )
+                pass
```

Comment:
I presume these were just spamming the logs?

Reply:
Yes. The logs are not helpful, but the checks have to be left in because when we removed them everything broke.